src/components/ViewLayoutStructure.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { SafeAreaView, View, StyleSheet, Text } from "react-native";
+import { Entypo, AntDesign } from "@expo/vector-icons";
+import TopPageStructure from "./TopPageStructure";
+
+export const ViewLayoutStructure = ({ children }) => {
+  return (
+    <SafeAreaView style={styles.container}>
+      <TopPageStructure />
+      <View style={styles.contentContainer}>{children}</View>
+      <View style={styles.menuContainer}>
+        <View style={styles.menuItem}>
+          <AntDesign
+            name="staro"
+            size={24}
+            color="black"
+            style={styles.menuIcon}
+          />
+          <Text style={styles.menuText}>Favourite</Text>
+        </View>
+        <View style={styles.menuItem}>
+          <AntDesign
+            name="team"
+            size={24}
+            color="black"
+            style={styles.menuIcon}
+          />
+          <Text style={styles.menuText}>Organizations</Text>
+        </View>
+        <View style={styles.menuItem}>
+          <AntDesign
+            name="calendar"
+            size={24}
+            color="black"
+            style={styles.menuIcon}
+          />
+          <Text style={styles.menuText}>Calendar</Text>
+        </View>
+        <View style={styles.menuItem}>
+          <Entypo
+            name="text-document"
+            size={24}
+            color="black"
+            style={styles.menuIcon}
+          />
+          <Text style={styles.menuText}>Feed</Text>
+        </View>
```

Comment:
wydziel to do komponentu jakiegos i mozna zrobic w /components np. folder navigation albo cos innego